index.html

```diff
@@ -1429,6 +1429,27 @@ <h4>Text</h4>
           <p>The source language of <a>Translation</a> <a>Text</a> objects and, where applicable,
             <a>Original</a> <a>Text</a> objects
             is indicated using the <a>Text Language Source</a> property.</p>
+          <p>If the <a>Text</a> object, or part of it, represents something more specific, or different,
+            to its parent <a>Script Event</a>, it can have a different <a>Represents</a> property.</p>
```

Comment:
I don't think we should encourage the Text object itself to have a different "represents" than the parent. 
```suggestion
          <p>If a part of a <a>Text</a> object represents something more specific, or different,
            to its parent <a>Script Event</a>, it can have a different <a>Represents</a> property.</p>
```

Reply:
Sorry for the confusion. There are 2 separate topics:
1. I think we should create a new issue to mention the case "if the semantic structure of the script is to have multiple paragraphs in the same Script Event". That is not obvious to me.
2. I think my initial suggestion becomes irrelevant.